### DISCUSSION TITLE
Hide within the app, except for admins

### DISCUSSION THREAD

**Discussion by Christilut** [directus/directus#12481] [Feature Requests]
I manage a Directus instance for a client. Settings for the project are also managed in Directus which makes it very nice as an easy/basic admin UI.
Some collections (like dangerous technical settings) are hidden from the normal users. I know I can do that with permissions and that works fine.

The problem occurs with M2M collections. Normal users need to be able to add/edit/view them inside their related collections. If I turn off the view permission, then inside the related collection, that field becomes invisible. The internal M2M collections need to be hidden so in order to achieve that, I have to use "Hide within the app", which is enabled by default for these M2M collections.

Now when I want to delete something from those hidden M2M collections, I have to uncheck "Hide within the app", then delete the items and re-check it. It happens fast so I'm not worried a normal user messes with the M2M collection in the meanwhile but that is also a smaller concern.

So as admin I have to go through a few extra steps when I want to delete user hidden content, it would be nice if this would be a bit more streamlined.

Maybe it is an idea to add a checkbox that still lets admin users see all collections (maybe with a special color) regardless if they are hidden? Or another solution for this problem?

**Comment by azrikahar** [comment]
Provided I understood this correctly, you can right click on the navigation bar (as an admin) to view hidden collections 👍

![chrome_AIDuQULufB](https://user-images.githubusercontent.com/42867097/160830113-12312bfc-535f-4bca-a1f1-35101f531e6c.gif)

**Reply by benhaynes** [reply]
As for hidden fields, I think we could do something similar on the item form.

**Reply by Christilut** [reply]
Oh wow, I didn't know about the right click. That's exactly what I needed!

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
